docs/core/linux-prerequisites.md

```diff
@@ -280,31 +346,41 @@ curl -sSL -o dotnet.tar.gz https://go.microsoft.com/fwlink/?linkid=848821
 ```
 
 ---
-3. Extract the .NET Core SDK binaries.
+
+4. Extract the .NET Core SDK binaries.
+
     ```bash
     sudo mkdir -p /opt/dotnet && sudo tar zxf dotnet.tar.gz -C /opt/dotnet
     ```
-4. Add dotnet to your PATH.
+
+5. Add dotnet to your PATH.
+
     ```bash
     sudo ln -s /opt/dotnet/dotnet /usr/local/bin
     ```
-5. Run the `dotnet --help` command to prove the installation succeeded.
+
+6. Run the `dotnet --help` command to prove the installation succeeded.
 
      ```bash
      dotnet --help
      ```
+
 ## Install .NET Core for SUSE Linux Enterprise Server (64 bit) (.NET Core 2.x) and openSUSE (64 bit)
 
+To Install .NET Core for SUSE Linux Enterprise Server (SLES) 12 SP2 (64 bit) and openSUSE 42.2 on NET Core 2.x, or openSUSE 42.1(64 bit) on .NET Core 1.x:
```

Comment:
Bad choice of words.  .NET Core for SUSE Linux Enterprise Server is only available for .NET Core 2.0.